resources/charlie/chainspec.toml

```diff
@@ -2,10 +2,10 @@
 # Human readable name for convenience; the genesis_hash is the true identifier.  The name influences the genesis hash by
 # contributing to the seeding of the pseudo-random number generator used in contract-runtime for computing genesis
 # post-state hash.
-name = 'casper-charlienet'
+name = 'casper-charlie-testnet'
 # Timestamp for the genesis block, also used in seeding the pseudo-random number generator used in contract-runtime for
 # computing genesis post-state hash.
-timestamp = 1600218437000
+timestamp = 1600452000000
```

Comment:
That's 6 PM GMT, right? (I thought I read 8 PM somewhere.)

Reply:
Yes, I was updating the old description from when I thought we might start at 20:00 yesterday.